3-networks/commons.auto.example.tfvars

```diff
@@ -26,7 +26,5 @@ domain = "example.com."
 
 access_context_manager_policy_id = 000000000000
```

Comment:
`access_context_manager_policy_id = 000000000000` is not used by the `shared` folder, so we might see the same error there. I think you will have to create at least three files:

**`commons.auto.example.tfvars` with this content:**
```
org_id = "000000000000"
terraform_service_account = "org-terraform@example-project-2334.iam.gserviceaccount.com"
default_region1 = "us-central1"
default_region2 = "us-west1"
domain = "example.com."
//Optional - for development.  Will place all resources under a specific folder instead of org root
//parent_folder = "000000000000"

```

**`dev_nonprod_prod.auto.example.tfvars` with this content:**
```
access_context_manager_policy_id = 000000000000
```

**`shared.auto.example.tfvars` with this content:**
```
target_name_server_addresses = ["8.8.8.8", "8.8.8.4"] // Only used in shared
```


What I don't know is if the `tf-wrapper.sh` will need to be changed as a result?

Reply:
The script is not affected by the number of tfvars files, it is only important that the symlinks point to the correct top level file.

I would suggest `access_context.auto.example.tfvars`  instead of  `dev_nonprod_prod.auto.example.tfvars`

Reply:
Done following Daniel's suggestion